### PR TITLE
Import Instruction added to readme.md

### PR DESCRIPTION
**Pull request by raxraj**
Solved #813 

Hey Managers,
I hope you all are safe and doing great things.
Just added a Usage option in the readme.md for the import statement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@
 npm install --save normalize.css
 ```
 
+**Usage**
+```sh
```
Comment (bronzehedwick) [inline README.md]:
```suggestion
```js
```

**Comment by raxraj** [comment]
@bronzehedwick I see what you have done there!
Thanks :)

**Comment by raxraj** [comment]
@bronzehedwick Haven't merged it Yet? 
Any issue?

File: README.md [hunk 2]
```diff
@@ -17,6 +17,11 @@
 npm install --save normalize.css
 ```
 
+**Usage**
+```js
+//using ES6
+import 'import normalize.css';
```
Comment (tniezurawski) [inline README.md]: That doesn't look valid. Is the name of imported package "import normalize.css"?

**Comment by bronzehedwick** [comment]
@raxraj I do not have merge access here.

**Pull request closed without merging**